src/main/native/jgskit.win64.cygwin.mak

```diff
@@ -11,9 +11,7 @@
 TOPDIR = $(MAKEDIR)\..\..\..
```

Comment:
Seems like this file and a few other files should have their copyrights update to include 2024.

Reply:
Copyrights updated.